src/jobs/js_install_dependencies_job.yml

```diff
@@ -13,11 +13,13 @@ steps:
   - checkout
   - run:
       name: Install dependencies
-      command: yarn --immutable
+      command: |
```

Comment:
i think it would be cleaner to have a parameter called: `working_directory` and then before the `steps` set:

```
working_directory: ~/project/<< parameters.working_directory >>
```

this allows you to not take care of the paths everywhere but set once and let the rest as it was before